src/Components/List/Card.scss

```
@@ -0,0 +1,81 @@
+@import "../../Styles/common.scss";
+
+.Card {
+    display: list-item;
+    position: relative;
+    width: 255px;
+    height: 383px;
+    margin-right: 20px;
+
+    .thumnailWrap {
+        position: relative;
+        width: 255px;
+        height: 255px;
+        border-radius: 6px;
+        overflow: hidden;
+
+        .imgWrap {
+            position: absolute;
+            top: 0px;
+            left: 0px;
+            right: 0px;
+            bottom: 0px;
+            width: 100%;
+            padding-top: 100%;
+            background: rgba(0, 0, 0, 0.02);
+            z-index: 0;
+        }
+
+        img {
+            width: 100%;
+            height: auto;
+            z-index: -1;
+        }
+    }
+
+    p {
+        font-size: 16px;
+
+        &.productName {
+            margin-top: 10px;
+            margin-bottom: 5px;
+            padding-right: 32px;
+            color: #777777;
+            font-weight: 400;
+        }
+
+        &.productPrice {
+            font-weight: 700;
+        }
+
+        &.productDiscountPrice {
+            color: $point;
+            font-weight: 700;
+        }
```

```suggestion
        .bold {
            font-weight: 700;
        }

        &.productDiscountPrice {
            color: $point;
        }
```
font 두께를 설정해주는 별도 클래스를 부여하면 중복을 줄일 수 있지 않을까요?

---

```
    &.productDiscountPrice {
      color: $point;
    }

    &.bold {
      font-weight: 700;
    }
```
요렇게 수정했습니다!